packages/engine-core/src/binary/Connection.ts

```diff
@@ -54,8 +54,7 @@ export class Connection {
     if (this._pool) return
 
     this._pool = new Pool(url, {
-      connections: 100,
-      pipelining: 10,
```

Comment:
The reason for this change is that we cannot pipeline request. In other words, we cannot make requests serial. Why? Because if you have - for instance - a transaction A that depends on B, transaction A will never finish if sent in the wrong order (as the http client waits for a response from the QE).